modules/special-rules/nav.adoc

```diff
@@ -18,43 +18,60 @@
   *** xref:vortex-missile.adoc[]
 
  ** xref:special-abilities.adoc[]
-  *** xref:anti-personnel-ability.adoc[]
-  *** xref:artillery.adoc[]
+
+ ** Special abilities: unit design modifiers
+  *** xref:armour-1.adoc[]
   *** xref:assault.adoc[]
-  *** xref:bridge.adoc[]
-  *** xref:brood-telepathy.adoc[]
-  *** xref:cavalry.adoc[]
-  *** xref:close-support.adoc[]
-  *** xref:deathwing.adoc[]
-  *** xref:engineers.adoc[]
+  *** xref:assault-1.adoc[]
   *** xref:fast.adoc[]
-  *** xref:flak.adoc[]
+  *** xref:firepower-1.adoc[]
   *** xref:heavy-weapons.adoc[]
   *** xref:hero.adoc[]
-  *** xref:hit-and-run.adoc[]
+  *** xref:long-range.adoc[]
+  *** xref:self-propelled-artillery.adoc[]
+  *** xref:shootier.adoc[]
+  *** xref:siege.adoc[]
+
+ ** Special abilities: unit abilities
+  *** xref:artillery.adoc[]
+  *** xref:cavalry.adoc[]
+  *** xref:close-support.adoc[]
+  *** xref:flak.adoc[]
+  *** xref:hitch-x.adoc[] *NEW* equivalent of Transport (X)
+  *** xref:holo-fields.adoc[]
+  *** xref:ignore-terrain.adoc[]
   *** xref:infiltrators.adoc[]
   *** xref:jump-packs.adoc[]
   *** xref:kroot.adoc[]
-  *** xref:long-range.adoc[]
-  *** xref:medic.adoc[]
-  *** xref:open-top.adoc[]
-  *** xref:ordo-malleus.adoc[]
   *** xref:psyker.adoc[]
   *** xref:rampage.adoc[]
   *** xref:rapid-fire.adoc[]
-  *** xref:ravenwing.adoc[]
-  *** xref:recovery.adoc[]
   *** xref:save.adoc[]
-  *** xref:self-propelled-artillery.adoc[]
-  *** xref:shootier.adoc[]
-  *** xref:siege.adoc[]
   *** xref:skimmer.adoc[]
-  *** xref:space-wolves.adoc[]
-  *** xref:stubborn.adoc[]
   *** xref:supreme-commander.adoc[]
-  *** xref:tank-destroyer.adoc[]
+  *** xref:synapse-creature.adoc[]
   *** xref:transport-x.adoc[]
   *** xref:tunneller.adoc[]
+  *** xref:war-engine-supreme-commander.adoc[] *NEW* this is just the ability to re-roll when supporting, for Dominatrixes and maybe add to Gargant Warlords
+
+ ** Special abilities: detachment abilities
+  *** xref:a-dying-race.adoc[]
+  *** xref:brood-telepathy.adoc[]
+  *** xref:dodge.adoc[]
+  *** xref:dauntless.adoc[]
+  *** xref:hit-and-run.adoc[]
+  *** xref:no-retreat.adoc[]
+  *** xref:psychic-beacon.adoc[]
+  *** xref:ravenwing.adoc[]
+  *** xref:regroup.adoc[]
+  *** xref:stubborn.adoc[]
+  *** xref:summon.adoc[] *NEW* updated name for 'summoning daemons'
+  *** xref:synapse.adoc[]
+  *** xref:unbreakable.adoc[]
+
+ ** Special abilities: army abilities
```

Comment:
I agree that it is useful to think about these four 'levels' of rules — unit datasheet, unit abilities, detachment abilities, army rules.
But the first and last aren't really 'abilities' are they? I think we'll need to adjust the titles and such for these, and perhaps not rely on the word ability/special ability. (Actually, I think you've mentioned something somewhere already about how we present the unit data sheet stuff — I'll look out for that too.)

But perhaps we can add some sort of note(s) somewhere to explain the relationship between these four things from the design point of view? This might help with conversations on Facebook etcetera?

Reply:
Do you mean the explanation of the four types of rule that's in special-abilities.adoc? Although I do need to tweak the intro text in that file, as it still refers specifically to units.

Also:
```suggestion
 ** Army rules
```

Reply:
Yes, I mean explanation of these four 'levels' of 'ability' rules.
Of course, `special-abilities.adoc` is set to be redundant, as it won't have any 'child' topics given that we're putting the new 'category parent' topics at the same nav level.
So, I think we want to re-use any useful bits of that topic, and create new content as appropriate in the four new parent topics.
Then -- if there is much to say about it -- perhaps we should have a module for design commentary, and we can link to a topic there from a note in each of the four new categories. Probably we should treat this last bit as outside the scope of this PR, and create a new issue if we think it would be a worthwhile exercise after we see how the four new parent topics look.

As for 'Army rules' let's go with it. We can always change it if we think of something that is more specific but not a mouthful.

Finally, while it is useful to rough this stuff out in one place, we'll want to move the 'Army rules' stuff to the Armies section won't we?

Reply:
> Of course, `special-abilities.adoc` is set to be redundant, as it won't have any 'child' topics given that we're putting the new 'category parent' topics at the same nav level.

Personally, I'm happy having one special abilities page and then nesting the sub-types below it in the navigation. Four levels of navigation is still pretty mild in wargaming terms, and having four separate description pages is going to involve a lot of duplication, while removing our 'special abilities' term for referring to all of them at once. Which is going to make it substantially harder to talk about.

> Finally, while it is useful to rough this stuff out in one place, we'll want to move the 'Army rules' stuff to the Armies section won't we?
Not that I'm aware of. Part of the point of classifying **all** the abilities like this was to collect them together in one place. If we move the Army rules into a different section we then run into a bunch of problems with the Unit and Detachment abilities because a bunch of them are race-specific like Holo-fields or A Dying Race, but aren't army-level rules.

Reply:
I try not to go more than three deep if possible, but it isn't always possible. There is more than one way to potentially achieve that here, but for now let's go with nesting as you suggest. Can always revisit later.

Re moving the army stuff: There's a difference between 'army-specific rules' and 'army-level rules', and I haven't had time to look at the build and think it through entirely. Ignore this for now. We can always reorganise and/or duplicate with partials later.

Reply:
Adding the 'Faction abilities' suggestion here, to use the proper channels.

```suggestion
 ** Faction abilities
```